### DISCUSSION TITLE
a simple template project

### DISCUSSION THREAD

**Discussion by RichardEllicott** [migueldeicaza/SwiftGodot#638] [Ideas]
i am totally new to swift, but i managed to figure out write gdextension in c++, and feed the audio buffer no simple stuff but i hate that header file boilerplate with a vengance (so am not interested in rust)... so i figure if i can get bootstrapped i will enjoy swift

however i can't work this out, please can we have a simple example, a spinning cube one or something like you have in the docs i think

maybe it's because you're using mac? i'm on windows and use vs.code... i dunno is there a crossplatform template making issue

**Comment by migueldeicaza** [comment]
There is a brand new template project listed:

https://github.com/elijah-semyonov/SwiftGodotTemplate
